app/views/giving/transactions/new.html.haml

```diff
@@ -0,0 +1,67 @@
+- @title = t('giving.heading')
+
+- content_for :js do
```

Comment:
I'm going to disagree in this instance. This JavaScript is so geared to this specific HTML that it doesn't make sense to break them at this point. Further, this requires a bit of injection that would be difficult to maintain for a static asset. If this is required for merging, I'll do it, but I strongly believe this will make the code more difficult to maintain.